src/planscape/plan/models.py

```diff
@@ -23,3 +24,43 @@ class Plan(models.Model):
 
     # The planning area of the plan.
     geometry = models.MultiPolygonField(srid=4269, null=True)
+
+class Project(models.Model): 
+    """
+    A Project is associated with one User, the owner, and one Plan. It has optional user-specified
+    project parameters, e.g. constraints.
+    """
+    owner = models.ForeignKey(User, on_delete=models.CASCADE, null=True) 
+
+    plan = models.ForeignKey(Plan, on_delete=models.CASCADE) 
+
+    # Project Paramaters
+
+    # The maximum cost constraint. Default set to no max cost.
+    max_cost = models.IntegerField(null=True, default=math.inf) 
```

Comment:
do we want null = no max cost or inf to = no max cost?

Reply:
Good question. Let's do null = no max cost.